src/text-tags.js

```diff
@@ -73,12 +73,13 @@ function buildTagsFromTopic(config, tag, url = {}) {
   return topicMetaData;
 }
 
-function buildTagsFromAuthor(config, author, url = {}) {
+function buildTagsFromAuthor(config, author, url = {}, data) {
   if(isEmpty(author)) return;
 
-  const authorName = author.name;
+  const customSeo = get(data,["data","customSeo"], {})
+  const authorName = customSeo.title || author.name;
   const authorUrl = `${config['sketches-host']}${url.pathname}`;
-  const description = author.bio || `View all articles written by ${authorName}`;
+  const description = customSeo.description || author.bio || `View all articles written by ${authorName}`;
```

Comment:
how are we going to differentiate in the app level for different seo tags if every key is a title/description, etc?

Reply:
??

Reply:
I'm differentiating it with a normal description will have a description as a keyword while ogDescription should have a description as a key. here is [doc](https://developers.quintype.com/malibu/tutorial/custom-seo-metadata.html)